city_scrapers/spiders/det_human_rights.py

```diff
@@ -0,0 +1,23 @@
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.spiders import CityScrapersSpider
+
+from city_scrapers.mixins import DetCityMixin
+
+
+class DetHumanRightsSpider(DetCityMixin, CityScrapersSpider):
+    name = 'det_human_rights'
+    agency = 'Detroit Human Rights Commission'
+    agency_cal_id = '1656'
+    agency_doc_id = []  # No docs at time of creation
+
+    def _parse_title(self, response):
+        title = super()._parse_title(response)
```

Comment:
The "- Meeting" doesn't add much, so I think just returning "Human Rights Commission" works here